src/components/completer-cmp.ts

```diff
@@ -135,6 +135,7 @@ export class CompleterCmp implements OnInit, ControlValueAccessor, AfterViewChec
     @Input() public openOnFocus = false;
     @Input() public initialValue: any;
     @Input() public autoHighlight = false;
+    @Input() public forceSelection = false;
```

Comment:
`forceSelection` should also be used in the component html template `[forceSelection]="forceSelection"`

Reply:
I made a confusion thinking completer-cmp-md.html was the template. I was rushing at that time ×)